torch/testing/_internal/common_methods_invocations.py

```diff
@@ -177,6 +177,7 @@ def __init__(self,
                  handles_extremals=True,  # whether the op correctly handles extremal values (like inf)
                  handles_complex_extremals=True,  # whether the op correct handles complex extremals (like inf -infj)
                  promotes_integers_to_float=False,  # whether op promotes unary output to float or not
+                 is_sparse_supported=False,
```

Comment:
"is_sparse_supported" -> "supports_sparse"

Reply:
This metadata should be on the OpInfo class, not just the subclass of UnaryUfuncInfo.

Reply:
This is done